src/ui/server/controllers/insurance/policy/index.test.ts

```diff
@@ -0,0 +1,73 @@
+import { TEMPLATE, pageVariables, get } from '.';
+import { PAGES } from '../../../content-strings';
+import { TEMPLATES } from '../../../constants';
+import { INSURANCE_ROUTES } from '../../../constants/routes/insurance';
+import insuranceCorePageVariables from '../../../helpers/page-variables/core/insurance';
+import getUserNameFromSession from '../../../helpers/get-user-name-from-session';
+import { Request, Response } from '../../../../types';
+import { mockApplication, mockReq, mockRes } from '../../../test-mocks';
+
+const {
+  INSURANCE_ROOT,
+  ALL_SECTIONS,
+  POLICY: { TYPE_OF_POLICY },
+  PROBLEM_WITH_SERVICE,
+} = INSURANCE_ROUTES;
+
+describe('controllers/insurance/policy/index', () => {
```

Comment:
do we need /index here?

Reply:
I think so yes because its a root index in the `policy` directory, indicates it's an index file in the root of `policy`. We have the same in the `business` directory and ~10 other files like this. Happy to change if desired.